simulation_ws/src/turtlebot3_description_reduced_mesh/models/turtlebot3_burger/model.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Can you please explain what this file is needed for? in general for all files that you're adding (that didn't exist in the ROS1 implementation in https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker) can you please detail what they're required / what we didn't need them in ROS1?

Reply:
Turtlebot3 community have already migrated turtlebot3 models for gazebo9 that we have added here as it is.
model.config is model config file in ROS2 that we have referred turtlebot3_simulations models from ROS2 branch as below link.

https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo/models/turtlebot3_burger